### PR TITLE
Implements 509 features since 509 is now officially stable.

### PR DESCRIPTION
**Pull request by Iamgoofball**
someone with merge access should fix the merge issues

:cl: Remie & MrPerson
tweak: Hit animations no longer colour or rotate with their targets
tweak: Progress bars no longer colour or rotate with their targets
/:cl:

### CODE REVIEW

**Comment by RemieRichards** [comment]
Conflicts already lol.

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ sudo: false
 
 env:
     global:
-    - BYOND_MAJOR="508"
-    - BYOND_MINOR="1293"
+    - BYOND_MAJOR="509"
+    - BYOND_MINOR="1300"
```
Comment (tkdrg) [inline .travis.yml]: this should be 1315

**Comment by MrPerson** [comment]
The matrices really doesn't need a changelog entry

**Comment by RemieRichards** [comment]
@MrPerson b-but pretty colours!

**Comment by MrPerson** [comment]
They don't add any pretty colors. Yet. Still gotta multiply them together and shit.

**Comment by MrStonedOne** [comment]
Roll out of 509 is halted, 509.1315 broke links in output

**Comment by Iamgoofball** [comment]
FUCK

**Comment by MrPerson** [comment]
Ok, the issue was resolved.

**Comment by RemieRichards** [comment]
@Iamgoofball conflicts

**Comment by Iamgoofball** [comment]
@RemieRichards buddy this is your guy's branch on the /tg/ repo, this isn't my problem to fix with conflicts

**Comment by MrStonedOne** [comment]
the 509 stable version has been removed from the pager updater, it is actually intentionally recommending downgrades

**Comment by Iamgoofball** [comment]
welp

**Comment by RemieRichards** [comment]
@Iamgoofball 
>you make a PR
>refuse to update it for conflicts

**Comment by MrStonedOne** [comment]
goofball doesn't have control over the from branch

**Comment by RemieRichards** [comment]
Then realistically, he shouldn't have made the PR!

**Comment by MrStonedOne** [comment]
well, literally anybody can fix the merge conflicts, by just doing a new pr to that branch

but only a maintainer could ever fix them directly

**Pull request closed without merging**